galaxykit/command.py

```diff
@@ -114,6 +116,14 @@ def main():
                         }
                     )
                     resp = users.update_user(client, user_data)
+
+                if subop == "remove":
+                    username, groupname = subopargs
+                    user_data = users.get_user(client, username)
+                    group_id = groups.get_group_id(client, groupname)
+                    user_data["groups"] = list(filter(lambda group: group['id'] != group_id, user_data["groups"]))
+                    pprint(user_data)
```

Comment:
this was probably meant for debugging, not merging?
(I see no other uses of `pprint` now, just a lot of imports)